pkg/healthcheck/healthcheck.go

```diff
@@ -49,3 +50,7 @@ func (s *State) Healthy() bool {
 func (c Controller) Route(w http.ResponseWriter, r *http.Request) {
 	api.SuccessResponse(w, Get())
 }
+
+func (c Controller) Info(w http.ResponseWriter, r *http.Request) {
+	api.SuccessResponse(w, settings.Get().Visibility)
```

Comment:
The `Visibility` is not enough information, create a specific struct which contains all the necessary fields and return one instance of it. You should simply call it `InfoResponse` and store the following field:
- Visibility
- Available oAuth providers (return with key-value pairs, name and availability)